feature/experimental/p4rt/ate_tests/base_p4rt/base_p4rt_test.go

```diff
@@ -407,7 +438,7 @@ func TestP4rtConnect(t *testing.T) {
 			},
 		})
```

Comment:
It is confusing that verifyReadReceiveMatch accepts a slice of expected_update but only uses the first one. Could you please change it so that verifyReadReceiveMatch accepts the single expected entity? Something like:

```
expected_updates := p4rtutils.ACLWbbIngressTableEntryGet([]*p4rtutils.ACLWbbIngressTableEntryInfo{{
	Type:          p4_v1.Update_INSERT,
	EtherType:     0x6007,
	EtherTypeMask: 0xFFFF,
	Priority:      1,
}})
expected_entity = expected_updates[0].Entity
if err := verifyReadReceiveMatch(t, expected_entity, readResp); err != nil {
  ...
```

Reply:
Hi Greg,

I will address this in the next commit.